INSTALL.md

```diff
@@ -397,6 +397,27 @@ To configure Synapse to expose an HTTPS port, you will need to edit
 For those of you upgrading your TLS certificate in readiness for Synapse 1.0,
 please take a look at [our guide](docs/MSC1711_certificates_FAQ.md#configuring-certificates-for-compatibility-with-synapse-100).
 
+## Email
+It is desirable for Synapse to have the capability to send email, particularly
+in the case of password reset.
+
+To configure an SMTP server for Synapse, modify the configuration section
+headed ``email``, and be sure to have at least the ``smtp_host``, ``smtp_port``
+and ``notif_from`` fields filled out. You may also need to set ``smtp_user``,
+``smtp_pass``, and ``require_transport_security``..
+
+If Synapse is not configured with an SMTP server, password reset via email will
+ be disabled.
+
+Previous versions of Synapse delegated the job of sending this email to an
```

Comment:
Previous to what? this paragraph is probably redundant in INSTALL.md anyway.

Reply:
modified to remove time reference,